### PR TITLE
Reintroduce market offer volume

### PR DESCRIPTION
**Pull request by dontrolle**

### CODE REVIEW

File: test/integration/market.integration.test.ts [hunk 1]
```diff
@@ -550,6 +556,7 @@ describe("Market integration tests suite", () => {
       const mockedMarket = mockito.spy(market);
       const semiBook = mockito.mock(Semibook);
       const ba = "asks";
+      const expectedGives = new Big(-12);
```
Comment (lnist) [inline test/integration/market.integration.test.ts:559]: testing with a negative value seems weird... gives is a uint

Comment (dontrolle) [reply test/integration/market.integration.test.ts:559]: Hmmm, I tend to agree. I didn't introduce those values (simply refactored the test), but I'll change it to something reasonable.

Comment (dontrolle) [reply test/integration/market.integration.test.ts]: Fixed in 46f17c50cda49ccd1d3f8b38f5ef9f7a4bbedad9

**Review by espendk — APPROVED** [review]
nice 👌

**Pull request merged**